README.md

```diff
@@ -10,7 +10,7 @@ You can find the deployed project at [🚫URL NAME GOES HERE](🚫copy and paste
 
 |                                                      [Michael Perez](https://github.com/Perezented)                                         |                                                       [Toricruz Mendiola](https://github.com/mtoricruz)                                       |
 | :-----------------------------------------------------------------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------------------------------------------------: |
-| [<img src="https://i.ibb.co/7yBt5rb/profpic.jpg" width = "200" />](https://github.com/Perezented)                                           |               [<img src="https://i.ibb.co/7yBt5rb/profpic.jpg" width = "200" />](https://github.com/mtoricruz)                                |
+| [<img src="https://i.ibb.co/JBq28bz/mperez.jpg" width = "135" />](https://github.com/Perezented)                                           |               [<img src="https://i.ibb.co/7yBt5rb/profpic.jpg" width = "200" />](https://github.com/mtoricruz)                                |
```

Comment:
This has broken github's checks.  It's part of the readme.md though. NPM start still works as well on this branch.